test/comparator/test_resources.py

```diff
@@ -64,19 +64,6 @@ def test_resources_change(self):
             "resource_database_v1beta1.proto",
         )
         self.assertEqual(file_resource_pattern_change.location.source_code_line, 13)
-        # A new file-level resource definition is added.
-        file_resource_addition = findings_map[
-            "A file-level resource definition `example.googleapis.com/t3` has been added."
-        ]
-        self.assertEqual(
-            file_resource_addition.category.name,
-            "RESOURCE_DEFINITION_ADDITION",
-        )
-        self.assertEqual(
-            file_resource_addition.location.proto_file_name,
-            "resource_database_v1beta1.proto",
-        )
-        self.assertEqual(file_resource_addition.location.source_code_line, 19)
```

Comment:
The resource `example.googleapis.com/t3` is not newly added, it's moved to message-level resource.